Author: "A. Maitland Bottoms" <bottoms@debian.org>
Subject: boost time conversion

Give boost::posix_time::seconds a 64bit integer since it seems no longer to
accept double as an initializer.

--- a/host/examples/rfnoc_rx_to_file.cpp
+++ b/host/examples/rfnoc_rx_to_file.cpp
@@ -188,7 +188,7 @@
 
     while (true) {
         if ((not first_lock_time.is_not_a_date_time()) and
-                (boost::get_system_time() > (first_lock_time + boost::posix_time::seconds(setup_time))))
+	    (boost::get_system_time() > (first_lock_time + boost::posix_time::seconds(long(setup_time)))))
         {
             std::cout << " locked." << std::endl;
             break;
@@ -202,7 +202,7 @@
         else {
             first_lock_time = boost::system_time();	//reset to 'not a date time'
 
-            if (boost::get_system_time() > (start + boost::posix_time::seconds(setup_time))){
+            if (boost::get_system_time() > (start + boost::posix_time::seconds(long(setup_time)))){
                 std::cout << std::endl;
                 throw std::runtime_error(str(boost::format("timed out waiting for consecutive locks on sensor \"%s\"") % sensor_name));
             }
--- a/host/examples/rfnoc_nullsource_ce_rx.cpp
+++ b/host/examples/rfnoc_nullsource_ce_rx.cpp
@@ -278,7 +278,7 @@
     std::cout << boost::format("Creating the USRP device with: %s...") % args << std::endl;
     uhd::device3::sptr usrp = uhd::device3::make(args);
 
-    boost::this_thread::sleep(boost::posix_time::seconds(setup_time)); //allow for some setup time
+    boost::this_thread::sleep(boost::posix_time::seconds(long(setup_time))); //allow for some setup time
     // Reset device streaming state
     usrp->clear();
     uhd::rfnoc::graph::sptr rx_graph = usrp->create_graph("rx_graph");
